main/src/main/java/org/mobilitydata/gtfsvalidator/notice/OverlappingFrequencyNotice.java

```diff
@@ -22,7 +22,8 @@ public OverlappingFrequencyNotice(
             "prevEndTime", prevEndTime.toHHMMSS(),
             "currCsvRowNumber", currCsvRowNumber,
             "currStartTime", currStartTime.toHHMMSS(),
-            "tripId", tripId));
+            "tripId", tripId),
+        SeverityLevel.WARNING);
```

Comment:
The spec uses `may`, so WARNING is correct here per RFC2119. 👍 

Although this notice is missing the severity designation in the Javadoc.

Reply:
This should be an ERROR.

Again, taking the GTFS reference and writing validator strictly based on it is like taking the Webster's *A Compendious Dictionary of the English Language* published in 1806 and saying that all words that don't exist in it are not proper English.

Reply:
The spec says "Multiple headways for the same trip are allowed, but may not overlap". "May not" should be considered as "must not", and treated as an ERROR.

Registered in Discrepancies Specification vs Validator MobilityData/gtfs-tasks#87

Reply:
This notice has been changed to `ERROR` in https://github.com/MobilityData/gtfs-validator/pull/726/commits/a5155d58b1a2be6cf996be4c8bc1b666cb845d14.